core/controllers/skill_editor_test.py

```diff
@@ -182,7 +186,16 @@ def test_editable_skill_handler_get_succeeds(self):
         self.login(self.ADMIN_EMAIL)
         # Check that admins can access the editable skill data.
         json_response = self.get_json(self.url)
-        self.assertEqual(self.skill_id, json_response['skill']['id'])
+        self.assertEqual(self.skill_id, json_response['skills'][0]['id'])
```

Comment:
Assert both skills

Reply:
This response only returns one skill. Test for multiple skills see below.